Makefile.src

```diff
@@ -68,6 +68,7 @@ TEST_PY = ${SRC_PY}
 TOOL_PY = \
 	wheel==0.33.1 \
 	setuptools==41.6.0 \
+	pip==9.0.1 \
```

Comment:
If we're going to pin pip (which is not a bad idea), why not the most recent instead of one that's 3+ years old?

Reply:
I remember I had some issues with rex.setup and novel pip versions...

Reply:
This will require non-trivial refactoring because newer pip ignores `setup.py` in favor of `pyproject.toml` when the latter is available. The codebase already has a couple of packages with `pyproject.toml`, but ignoring `setup.py` means `rex_init` and `rex_static` are also ignored, which breaks dependency and extension tracing in `rex.core`. That said, we can use moving to `pyproject.toml` as a good opportunity to refactor `rex.core` and get rid of `rex.setup`.

Reply:
> I remember I had some issues with rex.setup and novel pip versions...

`pip` deployed by default on ubuntu 19.10 includes a fairly old toml parser, which breaks on a key that starts with a digit.

The latest pip ignores `setup.py` when `pyproject.toml` exists, which breaks dependency and extension tracing in `rex.core`.

Reply:
Ah, fair enough. I guess we can leave this version for now, but we should try to have this refactor on our near-term to-do list. We could probably bundle this work with reorganizing the package installation process to use a frozen list of packages.

This stuff, plus the [improved dependency resolver in pip](https://pyfound.blogspot.com/2020/03/new-pip-resolver-to-roll-out-this-year.html) coming later this year should make things a bit more stable and/or "normal".